src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -189,7 +189,7 @@ public String t3XMP() {
                 + "</rdf:Seq></bibtex:editor>" + "<bibtex:bibtexkey>Clarkson06</bibtex:bibtexkey>"
                 + "<bibtex:journal>International Journal of High Fidelity</bibtex:journal>"
                 + "<bibtex:booktitle>Catch-22</bibtex:booktitle>" + "<bibtex:pdf>YeKis03 - Towards.pdf</bibtex:pdf>"
-                + "<bibtex:keywords>peanut,butter,jelly</bibtex:keywords>"
+                + "<bibtex:keywords>peanut, butter, jelly</bibtex:keywords>"
```

Comment:
To simplify the tests I kept it like that. If not, one would need to have two different "correct" results. As we use `, ` as separator at the moment it seems like an OK thing to do.